### ISSUE TITLE
bug with database names at step 4 of installation 

### ISSUE THREAD

**Issue by spacecabbie** [chamilo/chamilo-lms#5154]
**Describe**
when installing chamilo at step 4 there is a small old bug,
basicaly if you have a database named: username_chamilo-database (many hosting providers use user as pre tag)
when you check database connection or go next no error show and test is succsesfull, but it removed the - sign for the databasename field.

**To Reproduce**
fresh install and name the new database like username_chamilo-database 

**Expected behavior**
obviously it should not adjust the db name rather give and error or just procceed without change of name.

**Screenshots**
https://imgur.com/a/tfUXb5L

**Comment by ywarnier** [comment]
Yes. There's something about dashes (I think the `-` is the issue, not the `_`) that we don't accept. I don't remember why, but there was a reason for that.

**Comment by spacecabbie** [comment]
Yes it does seemed familiar was unsure if i ever made a bug report or if it was supposed to be fixed
